packages/extension/src/utils/tests/generateReportObject.ts

```diff
@@ -37,9 +37,12 @@ describe('generateReport', () => {
         message: 'Exempted Cookies',
       },
     });
+
     globalThis.fetch = jest.fn().mockResolvedValue({
       json: jest.fn().mockResolvedValue({}),
     });
+
+    //@ts-ignore
```

Comment:
Let's keep adding reasons.

Reply:
Fixed in [d2aca92](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/814/commits/d2aca929c6b25a68d828a6bd79e4e422e6b91e5f)